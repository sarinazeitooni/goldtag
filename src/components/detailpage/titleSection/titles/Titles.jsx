import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
import {faStar} from "@fortawesome/free-solid-svg-icons";
import titleSectionData from "../Data/titleSectionData";
import {BrowserView} from "react-device-detect";
import React from "react";

function Titles({id , titleSection}) {
    const stars = [1,2,3,4,5];
    return(
        <div>
            <h1 className='title'>{id.title}</h1>
            <h4 className='subtitle'>{id.subtitle}</h4>
            <div className='star-container'>
                <div className='stars'>
                    {stars.map(() => { return (<FontAwesomeIcon icon={faStar} />) })}
                    {titleSectionData.comments}
                </div>
                <BrowserView><div><button className='submit-btn'>{titleSection.shipping}</button></div></BrowserView>
            </div>
        </div>
    )
};
export default Titles;