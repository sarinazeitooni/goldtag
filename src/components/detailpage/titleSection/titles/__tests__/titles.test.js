import React from 'react';
import { shallow } from 'enzyme';
import Titles from "../Titles";
describe("Titles component" , ()=>{
    const id={title : '', subtitle : ''};
    const titleSection = {shipping : ''};
    const container = shallow(<Titles id={id} titleSection={titleSection}/>);
    it("it should render a div",()=>{
        expect(container).toMatchSnapshot();
    });
    it("it should render",()=>{
        expect(container.find('div.like-dislike-container')).toBeTruthy();
    });
});