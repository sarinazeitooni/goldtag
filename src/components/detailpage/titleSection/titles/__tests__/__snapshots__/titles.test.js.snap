// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titles component it should render a div 1`] = `ShallowWrapper {}`;
